test_scripts/Smoke/HeartBeat/ATF_HeartBeat_App_does_not_send_HB_but_respond.lua

```diff
@@ -0,0 +1,101 @@
+--  Requirement summary:
```

Comment:
@istoimenova 
I fixed this problem with "self.mobileSession.answerHeartbeatFromSDL = true" in [ATF does not respond to HB from SDL](https://adc.luxoft.com/jira/browse/SDLOPEN-1153) and now it works correctly